typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -818,9 +824,10 @@ def x(self): ...
             self.assertIsSubclass(C, P)
             self.assertIsSubclass(C, PG)
             self.assertIsSubclass(BadP, PG)
-            self.assertIsSubclass(PG[int], PG)
-            self.assertIsSubclass(BadPG[int], P)
-            self.assertIsSubclass(BadPG[T], PG)
+            if not PEP_560:
```

Comment:
Is there a similar regression for non-protocol generics? If so, fine. If not, this is a bit unfortunate (though it may be unavoidable).

Reply:
Yes, this is not different from normal generics. This is already mentioned in the PEP 560 as one of the few user visible changes.